### PR TITLE
rename ContainerName to EnvSourceContainerName

### PR DESCRIPTION
**Pull request by zroubalik**
Signed-off-by: Zbynek Roubalik <zroubali@redhat.com>

### CODE REVIEW

File: content/docs/2.0/concepts/scaling-jobs.md [hunk 1]
```diff
@@ -49,24 +50,27 @@ You can find all supported triggers [here](../scalers).
...
-    activeDeadlineSeconds: 600 # Specifies the duration in seconds relative to the startTime that the job may be active before the system tries to terminate it; value must be positive integer
-    backoffLimit: 6 # Specifies the number of retries before marking this job failed. Defaults to 6
+    parallelism: 1              # Max number of desired instances ([docs](https://kubernetes.io/docs/concepts/workloads/controllers/jobs-run-to-completion/#controlling-parallelism))
+    completions: 1              # Desired number of successfully finished instances ([docs](https://kubernetes.io/docs/concepts/workloads/controllers/jobs-run-to-completion/#controlling-parallelism))
+    activeDeadlineSeconds: 600  # Specifies the duration in seconds relative to the startTime that the job may be active before the system tries to terminate it; value must be positive integer
+    backoffLimit: 6             # Specifies the number of retries before marking this job failed. Defaults to 6
 ```
 
 The optional configuration parameter. Currently not implemented. It is going to be supported.
```
Comment (zroubalik) [inline content/docs/2.0/concepts/scaling-jobs.md:59]: @TsuyoshiUshio I don't understand this note? Is this some leftover that could be removed? Thanks

File: content/docs/2.0/concepts/authentication.md [hunk 2]
```diff
@@ -136,10 +136,10 @@ Every parameter you define in `TriggerAuthentication` definition does not need t
...
 
 ```yaml
-env: # Optional.
-  - parameter: region # Required - Defined by the scale trigger
-    name: my-env-var # Required.
-    containerName: my-container # Optional. Default: scaleTargetRef.containerName of ScaledObject
+env:                              # Optional.
+  - parameter: region             # Required - Defined by the scale trigger
+    name: my-env-var              # Required.
+    containerName: my-container   # Optional. Default: scaleTargetRef.envSourceContainerName of ScaledObject
```
Comment (tomkerkhove) [inline content/docs/2.0/concepts/authentication.md:142]:
```suggestion
    envSourceContainerName: my-container   # Optional. Default: scaleTargetRef.envSourceContainerName of ScaledObject
```

File: content/docs/2.0/concepts/authentication.md [hunk 3]
```diff
@@ -93,27 +93,27 @@ metadata:
...
-      path: {hasicorp-vault-secret-path} # Required.
+      provider: none | azure | gcp | spiffe | aws-eks | aws-kiam  # Optional. Default: none
+  secretTargetRef:                                    # Optional.
+  - parameter: {scaledObject-parameter-name}          # Required.
+    name: {secret-name}                               # Required.
+    key: {secret-key-name}                            # Required.
+  env:                                                # Optional.
+  - parameter: {scaledObject-parameter-name}          # Required.
+    name: {env-name}                                  # Required.
+    containerName: {container-name}                   # Optional. Default: scaleTargetRef.envSourceContainerName of ScaledObject
```
Comment (tomkerkhove) [inline content/docs/2.0/concepts/authentication.md:104]:
```suggestion
    envSourceContainerName: {container-name}          # Optional. Default: scaleTargetRef.envSourceContainerName of ScaledObject
```

**Review by tomkerkhove — APPROVED** [review]

File: content/docs/2.0/concepts/authentication.md (hunk 2 above)
Comment (zroubalik) [reply content/docs/2.0/concepts/authentication.md:142]: TriggerAuthentication has not been changed, the property there is still `containerName`. Imho it makes sense as the property in TA is not confusing as in ScaledObject.

Comment (tomkerkhove) [reply content/docs/2.0/concepts/authentication.md:142]: Frankly, I didn't check the file - Sorry about that 🙈

Comment (zroubalik) [reply content/docs/2.0/concepts/authentication.md:142]: Not a problem 😄 But if you think that we should change this for TA as well, we can consider that.

Comment (tomkerkhove) [reply content/docs/2.0/concepts/authentication.md:142]: Nah it's ok!

File: content/docs/2.0/concepts/scaling-jobs.md (hunk 1 above)
Comment (TsuyoshiUshio) [reply content/docs/2.0/concepts/scaling-jobs.md:59]: Sorry, Probably, It was my mistake remove it. :) I double checked if the PollingInterval is valid or not. It is valid.

**Pull request merged**